monitor/backends_stat_provider.cpp

```diff
@@ -42,7 +42,7 @@ static void fill_backend_backend(rapidjson::Value &stat_value,
                                  rapidjson::Document::AllocatorType &allocator,
                                  const struct dnet_backend_io *backend,
                                  const dnet_backend_info *config_backend) {
-	char *json_stat = NULL;
+	char *json_stat = nullptr;
```

Comment:
Changes in `monitor/backends_stat_provider.cpp` and `monitor/statistics.cpp` are not connected with bulk_read command. Move them into a separate commit at least and/or to a separate pull request at most.